components/landing-page/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "landing-page",
-  "description": "",
-  "version": "1.1.2",
+  "description": "The LandingPage microapp for the OpenComponents website",
```

Comment:
`microapp`? I would avoid introducing new naming we don't use everywhere else to avoid confusion, what you suggest?

Reply:
component

Reply:
amended